show/main.py

```diff
@@ -53,8 +53,13 @@ class InterfaceAliasConverter(object):
     def __init__(self):
         self.alias_max_length = 0
 
-        config_db = ConfigDBConnector()
-        config_db.connect()
+        try:
+            config_db = ConfigDBConnector()
+            config_db.connect()
+        except Exception as e:
```

Comment:
Can you provide a more specific Exception(s) here?

Reply:
will add it